### DISCUSSION TITLE
SDOH Condition.category

### DISCUSSION THREAD

**Discussion by XcrigX** [synthetichealth/synthea#1370] [General]
I am looking at some recent data I generated with Synthea and noticing that a high percentage of Conditions generated seems to be SDOH-related. 
These all have _Condition.category = encounter-diagnosis._ 

Primary question: 
Should these use a different Condition category (problem-list-item, health-concern, or one of the SDOH profile values)  ?

Secondary question: 
Is it realistic to have so much of this data? I generated 20k patients, and around 50% of the Conditions generated are not what OMOP/OHDSI consider to be in the SNOMED "condition" domain. I'd agree these factors are important, but it surprised me to see so much of it. 

Here is a listing of all the generated condition codes for one sample patient to illustrate the prevalence of these: 
```
224295006	Only received primary school education (finding) **
160903007	Full-time employment (finding) **
1.24171E+14	Chronic intractable migraine without aura
82423001	Chronic pain
196416002	Impacted molars
15777000	Prediabetes
162864005	Body mass index 30+ - obesity (finding)
73595000	Stress (finding) **
160903007	Full-time employment (finding) **
6525002	Dependent drug abuse (disorder) **
55680006	Drug overdose
160903007	Full-time employment (finding) **
73595000	Stress (finding) **
73595000	Stress (finding) **
160903007	Full-time employment (finding) **
314529007	Medication review due (situation)
237602007	Metabolic syndrome X (disorder)
73595000	Stress (finding) **
424393004	Reports of violence in the environment (finding) **
278860009	Chronic low back pain (finding)
1.121E+12	Chronic neck pain (finding)
59621000	Essential hypertension (disorder)
741062008	Not in labor force (finding) **
160903007	Full-time employment (finding) **
314529007	Medication review due (situation)
314529007	Medication review due (situation)
44054006	Diabetes mellitus type 2 (disorder)
55822004	Hyperlipidemia
160904001	Part-time employment (finding) **
73595000	Stress (finding) **
444814009	Viral sinusitis (disorder)
314529007	Medication review due (situation)
302870006	Hypertriglyceridemia (disorder)
160903007	Full-time employment (finding) **
195662009	Acute viral pharyngitis (disorder)
73595000	Stress (finding) **
314529007	Medication review due (situation)
414545008	Ischemic heart disease (disorder)
274531002	Abnormal findings diagnostic imaging heart+coronary circulat (finding)
399261000	History of coronary artery bypass grafting (situation)

```

**Comment by dehall** [comment]
You're right - we can do better here. Condition.category is only set when US Core is enabled (which it is by default) but we always set it to "encounter-diagnosis". That's fine for most of the Conditions in Synthea, but for these SDoH type conditions there are more appropriate options. From the US Core IG perspective, these could be “problem-list-item” or “health-concern” as you note, or from the SDOH Clinical Care (ie, Gravity) perspective these could fall into one of their categories:
educational-attainment
elder-abuse
employment-status
financial-insecurity
food-insecurity
health-insurance-coverage-status
health-literacy
homelessness
housing-instability
inadequate-housing
intimate-partner-violence
material-hardship
medical-cost-burden
social-connection
stress
transportation-insecurity
veteran-status

Because category is 0..* we could even use multiple at a time.

Things get a little tricky as we try to support different versions of US Core but we can definitely improve this.

**Reply by dehall** [reply]
Note I just opened #1371 to track this
